src/commands/math/advancedSymbols.js

```diff
@@ -2,43 +2,202 @@
  * Symbols for Advanced Mathematics
  ***********************************/
 
-LatexCmds.notin =
-LatexCmds.cong =
-LatexCmds.equiv =
-LatexCmds.oplus =
-LatexCmds.otimes = P(BinaryOperator, function(_, super_) {
-  _.init = function(latex) {
-    super_.init.call(this, '\\'+latex+' ', '&'+latex+';');
-  };
-});
-
-LatexCmds['≠'] = LatexCmds.ne = LatexCmds.neq = bind(BinaryOperator,'\\ne ','&ne;');
-
-LatexCmds.ast = LatexCmds.star = LatexCmds.loast = LatexCmds.lowast =
-  bind(BinaryOperator,'\\ast ','&lowast;');
-  //case 'there4 = // a special exception for this one, perhaps?
-LatexCmds.therefor = LatexCmds.therefore =
-  bind(BinaryOperator,'\\therefore ','&there4;');
-
-LatexCmds.cuz = // l33t
-LatexCmds.because = bind(BinaryOperator,'\\because ','&#8757;');
-
-LatexCmds.prop = LatexCmds.propto = bind(BinaryOperator,'\\propto ','&prop;');
-
-LatexCmds['≈'] = LatexCmds.asymp = LatexCmds.approx = bind(BinaryOperator,'\\approx ','&asymp;');
-
-LatexCmds.isin = LatexCmds['in'] = bind(BinaryOperator,'\\in ','&isin;');
+//spacing
+LatexCmds.quad = LatexCmds.emsp = bind(VanillaSymbol,'\\quad ','    ');
+LatexCmds.qquad = bind(VanillaSymbol,'\\qquad ','        ');
+/* spacing special characters, gonna have to implement this in LatexCommandInput::onText somehow
+case ',':
+  return VanillaSymbol('\\, ',' ');
+case ':':
+  return VanillaSymbol('\\: ','  ');
+case ';':
+  return VanillaSymbol('\\; ','   ');
+case '!':
+  return Symbol('\\! ','<span style="margin-right:-.2em"></span>');
+*/
 
-LatexCmds.ni = LatexCmds.contains = bind(BinaryOperator,'\\ni ','&ni;');
+//
+// Relation symbols (Sec 3.8)
+//
+var BINARY_OPERATOR = [
+// Relation Symbols
+'equiv:2261', 'prec:227a', 'succ:227b', 'sim:223c', 'perp:22a5', 
+'preceq:2aaf', 'succeq:2ab0','simeq:2243', 
+'mid:2223', 'll:226a', 'gg:226b', 'asymp:224d', 'parallel:2225',
+'bowtie:22c8', 'smile:2323', 'sqsubseteq:2291', 'sqsupseteq:2292', 'doteq:2250',
+'frown:2322', 'ni:220b', 'propto:221d', 'vdash:22a2', 'dashv:22a3', 'owns:220b',
+
+// Binary operators
+'mp:2213', 'ominus:2296', 'uplus:228e', 'sqcap:2293', 'ast:2217', 'sqcup:2294',
+'bigcirc:25ef', 'bullet:2219', 'ddagger:2021', 'wr:2240', 'amalg:2a3f',
+
+'cdot:22c5', 'circ:2218', 'div:00f7', 'pm:00b1', /* 'times:00d7', */ 'cap:2229',
+'cup:222a', 'setminus:2216', 'land:2227', 'lor:2228', 'wedge:2227',
```

Comment:
how are `\land` and `\wedge` distinct? Seems like they're the same class and the same character?